### PR TITLE
See PR Description

### PR DESCRIPTION
**Pull request by jacobduncan00**
- Admin Dashboard UI Update
- Sessions for admins
- .env update
- flask-seeder library added
- user seeder added
- changed table from user to users due to psql keyword
- update login form, removed create account and forgot password

### CODE REVIEW

File: api/__init__.py [hunk 1]
```diff
@@ -29,6 +33,12 @@
 dotenv_path = join(dirname(__file__), '.env')
 load_dotenv(dotenv_path)
 
+# Import for env file:
```
Comment (jtanderson) [inline api/__init__.py]: This looks redundant with the block above.

File: api/__init__.py [hunk 2]
```diff
@@ -39,19 +49,27 @@
 def create_app():
     # Initial configurations:
     CORS(app)
-    app.config['SECRET_KEY'] = ''
-    
+    app.config['SECRET_KEY'] = os.environ.get("SECRET_KEY")
```
Comment (jtanderson) [inline api/__init__.py:46]: What is the `SECRET_KEY` purpose? To prevent XSRF? If yes, when/how does it generated? By the `CORS(app)` call?

Comment (jacobduncan00) [reply api/__init__.py:46]: Secret key needs to be set in order to use sessions

Comment (jacobduncan00) [reply api/__init__.py:46]: For now I just have the user insert their own secret key in their env file

File: api/__init__.py (hunk 1 above)
Comment (jacobduncan00) [reply api/__init__.py]: This is indeed redundant, I will remove

File: api/views.py [hunk 3]
```diff
@@ -62,14 +69,25 @@ def contact():
...
 @views.route('/temp', methods=['GET'])
 def temp():
-    from api import db, UserModel
-    me = UserModel(username='jventura3', 
-                   email='jventura3@gulls.salisbury.edu',
-                   password='a900a10c4d321f8338325b3cea14cbaa53a87bd3462bac578ab3966ea7ab9db8')
+    from api import db, UsersModel
+    me = UsersModel(username='jventura3',
+                    email='jventura3@gulls.salisbury.edu',
+                    password='a900a10c4d321f8338325b3cea14cbaa53a87bd3462bac578ab3966ea7ab9db8')
```
Comment (jtanderson) [inline api/views.py]: Probably shouldn't commit this to the repository. Better to just put this creation code in a script somewhere (that gets ignored) so you can run it manually to add the user.

File: api/__init__.py (hunk 2 above)
Comment (jtanderson) [reply api/__init__.py:46]: If by you mean the server operator, that makes sense!

**Comment by jacobduncan00** [comment]
Fixed duplicate env lines in __init__.py

**Pull request merged**